compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Popup.skiko.kt

```diff
@@ -463,16 +472,6 @@ private val PopupProperties.platformInsets: PlatformInsets
         }
     }
 
-private fun Modifier.parentBoundsInWindow(
-    onBoundsChanged: (IntRect) -> Unit
-) = this.onGloballyPositioned { childCoordinates ->
```

Comment:
Why it's required to unroll this function? can we just replace the implementation inside?

Reply:
I'm not sure I follow. How exactly wrapping modifier creation in function causes extra compositions?

Reply:
I'd prefer keeping meaningfuly named functions over comments even if it's used once.
Probably not a blocker here